examples/wrapper-types/uniffi.toml

```diff
@@ -1,11 +1,58 @@
 [bindings.python]
 cdylib_name = "wrapper_types"
 
+[bindings.python.wrappers.JsonObject]
+imports = ["json"]
+wrap = "json.loads({})"
+unwrap = "json.dumps({})"
+
+[bindings.python.wrappers.WrappedUserAccount]
+imports = ["wrappers"]
+wrap = "wrappers.wrap_user_account({})"
+unwrap = "wrappers.unwrap_user_account({})"
+
 [bindings.ruby]
 cdylib_name = "wrapper_types"
 
 [bindings.swift]
 cdylib_name = "wrapper_types"
 
+[bindings.swift.wrappers.Url]
+type_name = "URL"
+imports = ["Foundation"]
+wrap = "URL(string: {})!"
+unwrap = "String(describing: {})"
+
+[bindings.swift.wrappers.WrappedUserAccount]
+type_name = "FlatUser"
+wrap = "wrapUserAccount({})"
+unwrap = "unwrapUserAccount({})"
+
 [bindings.kotlin]
 cdylib_name = "wrapper_types"
+package_name = "wrappertypes"
+
+[bindings.kotlin.wrappers.Url]
+imports = [ "java.net.URL" ]
+type_name = "URL"
+wrap = "URL({})"
+unwrap = "{}.toString()"
+
+[bindings.kotlin.wrappers.WrappedUserAccount]
+imports = [
+    "wrappers.FlatUser",
+    "wrappers.wrapUserAccount",
+    "wrappers.unwrapUserAccount",
+]
+type_name = "FlatUser"
+wrap = "wrapUserAccount({})"
+unwrap = "unwrapUserAccount({})"
+
+[scaffolding.wrappers.Handle]
+type_name = "Handle"
+
+[scaffolding.wrappers.JsonObject]
+type_name = "serde_json::Value"
```

Comment:
hrm - I think this answers my question above, but I don't think I like it. IIUC, unlike the "foreign" side of the world, it would be literally impossible to change this to anything else, because all the rust side of the world must already use this.  IOW, the reason I like the foreign types being in this .toml is that they *can* change the type, but it's literally impossible for the Rust side, right?

Reply:
I like the symmetry here. However, I'm confused where the unwrap/wrap expressions are for the scaffolding.

> but it's literally impossible for the Rust side, right?

I'm 95% in a agreement with this, but in the context of `impl`/`dyn` and `type` aliasing, I'm less sure.

Reply:
Having this in the .toml is like saying "this option exists in the '.toml tree' for maximum configurability. However, it is impossible to configure it." :)